src/adagrams.js

```diff
@@ -1,8 +1,133 @@
+// import { strict } from "assert";
+
 const Adagrams = {
+//WAVE 1:
   drawLetters() {
-    // Implement this method for wave 1
+// Set up hash of letters w/ frequency 
+    const letterFrequencies = {
+      'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3,
+      'H': 2, 'I': 9, 'J': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8,
+      'P': 2, 'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4, 'V': 2,
+      'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+    };
+// WAVE 1 -->
+// use entries 
+// creating the empty lettersPool array 
+    const lettersPool = [];
+// loop through frequencies to push letters to lettersPool 
+// entries will be an array of arrays w/ letter + count in each array 
+    const entries = Object.entries(letterFrequencies);
+    entries.forEach(entry => {
+      letter = entry[0];
+      count = entry[1];
```

Comment:
Two things:

1. Using `Object.entries` here gets the job done, but it's not super readable code. I might do something with `Object.keys` instead:
    ```js
    Object.keys(letterFrequencies).forEach(letter => {
      const count = letterFrequencies[letter];
      // ...
    ```

2. You're defining `letter` and `count` as global variables here! You should prefix them with `const` (why not `let`?)